src/util/typeGuards.ts

```diff
@@ -86,6 +110,57 @@ export function isWsFormattedFuturesUserDataEvent(
   return isWsFormattedUserDataEvent(data) && data.wsMarket.includes('usdm');
 }
 
+export function isWsSpotUserDataExecutionReportFormatted(data: WsFormattedMessage):
```

Comment:
Sorry to be a pain, can you change the order of the naming pattern to match the other methods? `isWsFormattedSpot******(`

Reply:
yep makes sense